docs/testing.rst

```diff
@@ -0,0 +1,92 @@
+Testing
+=======
+
+Testing is done using `Selenium WebDriver`_ for automating browser tests.
+`Tox`_ is used to execute testing procedures.
+
+Before running all tests locally, `ChromeDriver`_ (for testing on Chrome browser)
+and `GeckoDriver`_ (for testing on Firefox browser) are required.
+Currently `ChromeDriver v75.0.3770.8`_ and `GeckoDriver v0.24.0`_ are being used for the tests.
```

Comment:
This is probably too much detail for a high level document like this.

Reply:
I have removed this line and pushed the changes.